build-system/tasks/test.js

```diff
@@ -51,17 +51,10 @@ function getConfig() {
   return extend(obj, karmaConfig.default);
 }
 
-var prerequisites = ['build'];
-if (process.env.TRAVIS) {
-  // No need to do this because we are guaranteed to have done
-  // it.
-  prerequisites = [];
-}
-
 /**
  * Run tests.
  */
-gulp.task('test', 'Runs tests', prerequisites, function(done) {
```

Comment:
slightly worried with this. most of our old tasks sometimes don't complete before running the next task which is why I haven't been composing them. (a lot have combined and sync/async usage w/o returning the stream or promise). if it consistently works then im ok with it